.infra/index.ts

```diff
@@ -210,6 +216,28 @@ if (isAdhocEnv) {
         'prometheus.io/port': '9464',
       },
     },
+    {
+      nameSuffix: 'personalized-digest',
```

Comment:
We don't really need it for local env right? We can remove it I believe

Reply:
For testing digest locally we would need it, feels natural to have it ready?

Reply:
I just feel it will be a waste of local resources 99.99% of the time

Reply:
Enable it through separate local env variable?

Reply:
Yes we can, or just comment and enable manually. Whatever works for you :)